refactor(docs)!: remove font families from public tokens (#988)
Vonage/vivid-3: docs/design/design-tokens.md

### Typography

#### Typefaces

Set the font custom properties of your choice (or all) to override the default typography styles with your own.

```css
--vvd-typography-headline
--vvd-typography-subtitle
--vvd-typography-heading-1
--vvd-typography-heading-2
--vvd-typography-heading-3
--vvd-typography-heading-4
--vvd-typography-base
--vvd-typography-base-bold
--vvd-typography-base-code
--vvd-typography-base-condensed
--vvd-typography-base-condensed-bold
--vvd-typography-base-extended
--vvd-typography-base-extended-bold
```

```html preview
<style>
  :root {
  --vvd-typography-heading-4: 500 20px/1.4 Georgia;
  --vvd-typography-base-code: 400 16px Courier;
  --vvd-typography-base-bold: 500 16px Georgia;
  }
</style>
<vwc-header>

>   This header is set by an override '--vvd-typography-heading-4' 

Author: ```suggestion
  This header typeface is set by '--vvd-typography-heading-4' 
```

</vwc-header>
<pre>/*

>  * This block of code uses an override '--vvd-typography-base-code'

Author: ```suggestion
 * This block of code typeface is set by '--vvd-typography-base-code'
```

 */
</pre>

> <vwc-button appearance="filled" label="I using my override '--vvd-typography-base-bold'"></vwc-button>

Author: ```suggestion
<vwc-button appearance="filled" label="This button text is set by '--vvd-typography-base-bold'"></vwc-button>
```

### Typography

#### Typefaces

> Set the font css-variable to override the default typography with your font and typography scale.

Author: ```suggestion
Set the font custom properties of your choice (or all) to override the default typography styles with your own.
```

> #### Font families

Author: it's not font families anymore

Set the font-variable to override the typography tokens with your own font.

```css
--vvd-typography-headline
--vvd-typography-subtitle
--vvd-typography-heading-1
--vvd-typography-heading-2
--vvd-typography-heading-3
--vvd-typography-heading-4
--vvd-typography-base
--vvd-typography-base-bold
--vvd-typography-base-code
--vvd-typography-base-condensed
--vvd-typography-base-condensed-bold
--vvd-typography-base-extended
--vvd-typography-base-extended-bold
```

```html preview
<style>
  :root {

  --vvd-typography-heading-4: 500 20px/1.4 Georgia;

>   --vvd-typography-base-code: 400 14px Courier;

Author: makes more sense
```suggestion
  --vvd-typography-base-code: 400 14px monospace;
```

>   --vvd-typography-base-bold: 500 14px Georgia;

Author: change markup to refer to the tokens instead of font family
e.g. _"This header is set by `vvd-typography-heading-4`"_
